src/Assignment-2/Program.cs

```diff
@@ -0,0 +1,422 @@
+    internal class Program
+    {
+        private static bool _flag = false;
+
+        private static List<Contact> _contactList = new ();
+
+        /// <summary>
+        /// getFlag returns the _flag value
+        /// </summary>
+        /// <returns>return bool</returns>
+        public static bool GetFlag()
+        {
+            return _flag;
+        }
+
+        /// <summary>
+        /// Main class gets the operation to do and call the respective method
+        /// </summary>
+        /// <param name="args">Arguments</param>
+        public static void Main(string[] args)
+        {
+            short userInputInt;
+            Console.WriteLine("welcome!!");
+            while (GetFlag() != true)
+            {
+                Console.Clear();
+                Console.WriteLine("===========================================================");
+                Console.WriteLine("there are six operation here");
+                Console.Write("1.Enter new contact  ");
+                Console.Write("2.search contact  ");
+                Console.Write("3.Edit  ");
+                Console.Write("4.Delete  ");
+                Console.Write("5.View all  ");
+                Console.WriteLine("6.exit");
+                Console.WriteLine("choose any one to work on : ");
+                string? userInput = Input();
+
+                if (userInput != null)
+                {
+                    if (short.TryParse(userInput, out userInputInt))
+                    {
+                        NumericChoice(userInputInt);
+                    }
+                    else
+                    {
+                        StringChoise(userInput);
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// NumericChoise method is used to choose the choice as per the numeric
+        /// </summary>
+        /// <param name="userInputInt">input choice</param>
+        public static void NumericChoice(int userInputInt)
+        {
+            switch (userInputInt)
+            {
+                case 1:
+                    {
+                        Console.Clear();
+                        AddNewContact();
+                        break;
+                    }
+
+                case 2:
+                    {
+                        Console.Clear();
+                        SearchContact();
+                        break;
+                    }
+
+                case 3:
+                    {
+                        Console.Clear();
+                        EditContact();
+                        break;
+                    }
+
+                case 4:
+                    {
+                        Console.Clear();
+                        DeleteContact();
+                        break;
+                    }
+
+                case 5:
+                    {
+                        Console.Clear();
+                        int count = 0;
+                        foreach (var obj in _contactList)
```

Comment:
```suggestion
                        foreach (var contact in _contactList)
``` 
Variable name should be meaningful as per the usage.